docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -21,7 +21,7 @@ This section describes the *inner-loop* development workflow for Docker containe
 
 An application is composed of your own services plus additional libraries (dependencies). The following are the basic steps you usually take when building a Docker application, as illustrated in Figure 5-1.
 
-![The development process for Docker apps: 1 - Code your App, 2 - Write Dockerfile/s, 3 - Create images defined at Dockerfile/s, 4 - (optional) Compose services in the docker-compose.yml file, 5 - Run container or docker-compose app, 6 - Test your app or microservices, 7 - Push to repo and repeat. ](./media/image1.png)
+![Diagram showing the 7 steps it takes to create a containerized app.](./media/docker-app-development-workflow/life-cycle-containerized-apps-docker-cli.png)
```

Comment:
This would also be a good place to use the new image tag:

```markdown
:::image type="complex" source="<folderPath>" alt-text="<alt-text>":::
<long description here>
:::image-end:::
```

Using the original description as the long one in the tag.

The long description here gives context.